### PR TITLE
Comments: scroll to latest reply + ESCAPE to close

### PR DESCRIPTION
**Pull request by siddharthkp**
Also includes: https://github.com/codesandbox/codesandbox-client/pull/3866

- scroll to latest reply when comment is opened (after animations)
- hitting escape should close dialog

### CODE REVIEW

File: packages/common/src/utils/keycodes.ts [hunk 1]
```diff
@@ -8,3 +8,4 @@ export const ARROW_RIGHT = 39;
 export const ARROW_DOWN = 40;
 export const DOT = 190;
 export const BACKTICK = 192;
+export const ESCAPE = 27;
```
Comment (MichaelDeBoey) [inline packages/common/src/utils/keycodes.ts:11]: We already have `ESC`

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3867/commits/29a1afe0d783f858d4178bac9011e75388d6c092 is at https://pr3867.build.csb.dev/s/new.

File: packages/common/src/utils/keycodes.ts (hunk 1 above)
Comment (siddharthkp) [reply packages/common/src/utils/keycodes.ts:11]: oh duh, I missed it!

**Comment by CompuIves** [comment]
Oh yes!!! THIS IS PERFECT :D

**Pull request merged**